DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh

```diff
@@ -169,11 +169,34 @@ fi
 
 echo "[INFO (async_pass.sh)] envvars were set to TFDELAYSECONDS ${TFDELAYSECONDS} TIMEFRAME_RATE_LIMIT ${TIMEFRAME_RATE_LIMIT}"
 
+# root output enabled only for 2 permille of the cases
+# keeping AO2D.root QC.root o2calib_tof.root MFTAssessment.root mchtracks.root mchclusters.root
+SETTING_ROOT_OUTPUT="ENABLE_ROOT_OUTPUT_o2_mch_reco_workflow= ENABLE_ROOT_OUTPUT_o2_mft_reco_workflow= ENABLE_ROOT_OUTPUT_o2_tof_matcher_workflow= ENABLE_ROOT_OUTPUT_o2_aod_producer_workflow= ENABLE_ROOT_OUTPUT_o2_qc= "
+
+if [[ -z $RND_PERMILLE ]]; then
+  RND_PERMILLE=$((1 + $RANDOM % 2000))
+fi
+echo "Randomly generated number for current reco = $RND_PERMILLE"
+if [[ $RND_PERMILLE -ge 2 ]]; then
+  echo "ROOT ouput will be kept ONLY FOR SOME workflows"
+else
+  echo "ROOT ouput will be kept for ALL workflows"
+  SETTING_ROOT_OUTPUT+="DISABLE_ROOT_OUTPUT=0"
+fi
+echo SETTING_ROOT_OUTPUT=$SETTING_ROOT_OUTPUT
+
+# Enabling GPUs
+if [[ -n "$ALIEN_JDL_USEGPUS" ]]; then
+  echo "Enabling GPUS"
+  export GPUTYPE="HIP"
+  export GPUMEMSIZE=$((25 << 30))
+fi
+
 # reco and matching
 # print workflow
-IS_SIMULATED_DATA=0 WORKFLOWMODE=print DISABLE_ROOT_OUTPUT="" TFDELAY=$TFDELAYSECONDS NTIMEFRAMES=-1 SHMSIZE=16000000000 DDSHMSIZE=32000 ./run-workflow-on-inputlist.sh CTF list.list > workflowconfig.log
+env $SETTING_ROOT_OUTPUT IS_SIMULATED_DATA=0 WORKFLOWMODE=print TFDELAY=$TFDELAYSECONDS NTIMEFRAMES=-1 SHMSIZE=16000000000 DDSHMSIZE=32000 ./run-workflow-on-inputlist.sh CTF list.list > workflowconfig.log
```

Comment:
Does this work to pass all env variables with prepending `env`? I have not tested it

Reply:
env will set all the env vars that you have in front of the command.
If you use bash as shell, you could also use `eval`, which is a bash builtin instead of a separate binary like `env`.

Reply:
I read that `eval` can have issues. With `env` for me it works.

Reply:
But I would leave this as WIP till @catalinristea checks it for the randomization, and till we have a stable configuration for the EPNs.

Reply:
For the randomization, note that for now this is a kind of placeholder, since 1) we need to check how RANDOM behaves and 2) if we have less than 1000 we might have nothing.